app/components/signum/notification_drawer/component_controller.js

```diff
@@ -1,46 +1,48 @@
-import ApplicationController from "signum/controllers/application_controller"
+import ApplicationController from "signum/controllers/application_controller";
```

Comment:
Did you switch formatters? ; are left out most of the time now.

Reply:
i also use prettier, but i changed the semicolon setting the other day as some other project was not adding 😛 , will add it back